pkg/service/rtcservice.go

```diff
@@ -524,13 +524,6 @@ func (s *RTCService) startConnection(
 	var cr connectionResult
 	var err error
 
-	if !s.roomAllocator.CreateRoomEnabled() {
-		cr.Room, _, _, err = s.roomAllocator.CreateRoom(ctx, pi.CreateRoom, false)
```

Comment:
what would handle this room creation in this case?

Reply:
this is handled consistently by roommanager now